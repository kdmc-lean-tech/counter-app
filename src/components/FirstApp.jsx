import React, { Fragment } from 'react';
import PropTypes from 'prop-types';

const FirstApp = ({ name }) => {
  <Fragment>
    <h1>Hello, my name is { name }</h1>
  </Fragment>
}

FirstApp.propTypes = {
  name: PropTypes.string
}

export default FirstApp;
