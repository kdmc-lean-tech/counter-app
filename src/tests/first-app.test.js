import React from 'react';
import { shallow } from 'enzyme';
import FirstApp from '../components/FirstApp';

describe('FirstApp Test', () => {
  test('should to show <FirstApp / > success', () => {
    const name = 'Kevin';
    const wrapper = shallow(<FirstApp name={name}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
