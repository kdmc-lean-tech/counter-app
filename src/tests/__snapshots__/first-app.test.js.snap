// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstApp Test should to show <FirstApp / > success 1`] = `""`;
